articles/cdn/cdn-pop-list-api.md

```diff
@@ -32,7 +32,7 @@ To lock down your application to accept traffic only from Azure CDN from Microso
 
 Configure IP ACLing for your backends to accept traffic from Azure CDN from Microsoft's backend IP address space and Azure's infrastructure services only. 
 
-Use Azure Front Door [service tags](../virtual-network/service-tags-overview.md) with Azure CDN from Microsoft to configure Microsoft's backend IP ranges. For a complete list, see [IP Ranges and Service tags](https://www.microsoft.com/en-us/download/details.aspx?id=56519) for Microsoft services.
+Use AzureFrontDoor.Backend [service tags](../virtual-network/service-tags-overview.md) with Azure CDN from Microsoft to configure Microsoft's backend IP ranges. For a complete list, see [IP Ranges and Service tags](https://www.microsoft.com/en-us/download/details.aspx?id=56519) for Microsoft services.
```

Comment:
@hisasima please add "the" in front of AzureFrontDoor.Backend and remove the "s" off service tag.